onnxruntime/hosting/http/session.cc

```diff
@@ -44,6 +50,7 @@ void HttpSession::OnRead(beast::error_code ec, std::size_t bytes_transferred) {
   HandleRequest(std::move(req_));
 }
 
+// After writing, make the session read another request
```

Comment:
It's better to reuse the http session and it's access to the socket than to create a new one per request. All of the beast examples reuse their HTTP or websocket sessions in the same way because destroying them and creating new ones would be costly.